Domain/Domain/Sources/UseCase/ManageWhiteboardToolUseCase.swift

```diff
@@ -21,7 +21,12 @@ public final class ManageWhiteboardToolUseCase: ManageWhiteboardToolUseCaseInter
     }
 
     public func selectTool(tool: WhiteboardTool) {
-        currentToolSubject.send(tool)
+        let previousTool = currentToolSubject.value
+        if previousTool == tool {
+            currentToolSubject.send(nil)
+        } else {
+            currentToolSubject.send(tool)
+        }
```

Comment:
send(nil)을 통해 nil값에 따라 처리가 필요한 거 겠죠!?

`+`

삼항 연산자는 어떻게 생각하시나요??

```suggestion
let selectTool = currentToolSubject.value == tool ? nil: tool
currentToolSubject.send(selectTool)
```
이런 느낌으로!!

Reply:
깔끔해서 좋은 것 같습니다! 동~